specification/cognitiveservices/data-plane/ContentModerator/v1.0/ContentModerator.json

```diff
@@ -2219,24 +2945,27 @@
         },
         "matches": {
           "description": "The match details.",
-          "$ref": "#/definitions/MatchDetailProperties"
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Match"
+          }
         },
         "status": {
           "description": "The evaluate status",
-          "$ref": "#/definitions/ResponseStatus"
+          "$ref": "#/definitions/Status"
         }
       }
     },
-    "MatchDetailProperties": {
-      "description": "The match detail.",
+    "Match": {
+      "description": "The match details.",
       "properties": {
         "score": {
           "description": "Confidence score of the image match.",
           "type": "number"
         },
         "matchId": {
           "description": "The match id.",
-          "type": "string"
+          "type": "number"
```

Comment:
Why is the id a `number`?

Reply:
Match id type is defined as number in our service.